src/electron/build.action.mjs

```diff
@@ -25,18 +24,17 @@ import path from 'path';
 const ELECTRON_BUILD_DIR = 'build';
 
 export async function main(...parameters) {
-  const {platform, buildMode, stagingPercentage, publish} = getElectronBuildParameters(parameters);
-  const version = await getVersion(platform);
+  const {versionName, platform, buildMode, publish} = getElectronBuildParameters(parameters);
```

Comment:
Is publish always needed, or just for publication?
Can we move it out from getElectronBuildParameters if that's the case?
Let's extract directly from parameters. The build code shouldn't have to know about the publication if not needed.